lib/unimatrix/quartermaster/binary.rb

```diff
@@ -2,18 +2,16 @@ module Unimatrix::Quartermaster
 
   class Binary < Unimatrix::DynamicResource
```

Comment:
should this be inheriting from Resource now?

Reply:
Yes - I forgot to add that back! great catch